### PR TITLE
[Unity Bridge] Add support for Auth to unity-js-bridge

### PR DESCRIPTION
**Pull request by adam-maj**

### CODE REVIEW

File: packages/unity-js-bridge/src/thirdweb-bridge.ts [hunk 1]
```diff
@@ -130,7 +142,7 @@ class ThirdwebBridge implements TWBridge {
     }
     const walletInstance = this.walletMap.get(wallet);
     if (walletInstance) {
-      await walletInstance.connect({ chainId });
+      await walletInstance.connect(chainId);
```
Comment (joaquim-verges) [inline packages/unity-js-bridge/src/thirdweb-bridge.ts:145]: why the change here? I switched it to be an object coz there will be more options for other wallets

File: packages/unity-js-bridge/src/thirdweb-bridge.ts [hunk 2]
```diff
@@ -196,6 +208,15 @@ class ThirdwebBridge implements TWBridge {
       }
     }
 
+    if (addrOrSDK.startsWith("auth")) {
```
Comment (joaquim-verges) [inline packages/unity-js-bridge/src/thirdweb-bridge.ts:211]: you need to do the change in unity in `Wallet.cs` in the `Authenticate()` method, switch from sdk/auth/login to just auth/login

File: packages/unity-js-bridge/src/thirdweb-bridge.ts (hunk 1 above)
Comment (adam-maj) [reply packages/unity-js-bridge/src/thirdweb-bridge.ts:145]: It breaks the types & doesn't build for me
